testing-guidance/workflow-tests/ragdoll/ragdoll-authoring-improvements.md

```diff
@@ -0,0 +1,47 @@
+# Ragdoll Authoring Improvements Workflow Tests
+
+Testing in this area should focus on the new Ragdoll Authoring Improvements.
+
+## General Docs
+* [Creating and Simulating a PhysX Ragdoll](https://www.o3de.org/docs/user-guide/visualization/animation/animation-editor/creating-and-simulating-physx-ragdoll/)
+* [Animation Editor User Interface](https://www.o3de.org/docs/user-guide/visualization/animation/animation-editor/user-interface/)
+* [Setting up a Ragdoll](https://stabilization-22-10--o3deorg.netlify.app/docs/user-guide/visualization/animation/animation-editor/ragdoll/ragdoll-physics-setup/)
```

Comment:
As discuss offline, this is an link to stabilization document, which will have to be updated with the new link once O3DE 22.10 has been released.

@FuzzyCarterAWS @hultonha if there is any further guideline on how to proceed with these links, please comment.

Reply:
I've done the same with docs that haven't gone live yet.

Once the docs go live, I'll assign myself a task to go and verify all the docs are referencing the live version of the docs.

I'm unsure if the stabilization version of the docs will be deleted any time soon, so if they are alive for a time after the push them to dev, we shouldn't have any interruption to the docs.

Reply:
@moraaar @hultonha I'd advise resolving this with the knowledge that we'll have to update a few workflow docs once the next doc push goes live.

Reply:
Sounds good to me, thanks @FuzzyCarterAWS 👍